src/pages/General.vue

```diff
@@ -60,10 +60,15 @@ export default {
     ipcRenderer.on('settings-update', this.onSettingsUpdate);
     ipcRenderer.on('update-state', this.onUpdateState);
     ipcRenderer.send('update-state');
+    try {
+      this.$data.settings = await ipcRenderer.invoke('settings-read');
+    } catch (error) {
+      console.error(`settings-read() or failed with error ${ error }`);
+    }
     try {
       this.$data.version = await ipcRenderer.invoke('get-app-version');
     } catch (error) {
-      console.error(`get-app-version() failed with error ${ error }`);
+      console.error(`get-app-version() or failed with error ${ error }`);
```

Comment:
This change confuses me — why the addition of `or` here? The function call failed, right?

Reply:
Oh boy, that's was a ugly typo from my side :P
Thanks for catching it